cts/update/array/addToSet.json

```diff
@@ -0,0 +1,71 @@
+{
+  "1-number": {
+    "request": {
+      "update": "composites",
+      "updates": [
+        {
+          "q": {
+            "_id": "array"
+          },
+          "u": {
+            "$addToSet": {
+              "v": {
+                "$numberInt": "100"
+              }
+            }
+          }
+        }
+      ],
+      "$db": "{{.Database}}"
+    },
+    "response": {
+      "n": {
+        "$numberInt": "1"
+      },
+      "nModified": {
+        "$numberInt": "1"
+      },
+      "ok": {
+        "$numberDouble": "1.0"
+      }
+    }
+  },
+  "2-find": {
+    "request": {
+      "find": "composites",
+      "filter": {
+        "_id": "array"
+      },
+      "sort": {
+        "_id": {
+          "$numberInt": "1"
+        }
+      },
```

Comment:
A little over kill to sort by _id when we filter by _id so only one is returned. Same for others.

Reply:
Good point, have changed the filters instead. PTAL